po/extra/LINGUAS

```diff
@@ -0,0 +1,9 @@
+de
```

Comment:
Why does this file have a lot less LINGUAS than the other?

Reply:
Just because those are the only po files actually present in the data dir. In a later branch I can copy them and generate new po files so we have full coverage